src/screens/EventVolunteers/Requests/Requests.tsx

```
@@ -36,29 +36,28 @@
  * @example
  * <Requests />
  */
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { debounce } from '@mui/material';
 import { Button } from 'react-bootstrap';
```

ESLint

[error] 'Button' import from 'react-bootstrap' is restricted. Direct imports of Button from react-bootstrap are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)